apps/ledger-live-desktop/src/renderer/components/Web3AppWebview/helpers.ts

```diff
@@ -237,18 +243,13 @@ export function useWebviewState(
     webview.addEventListener("did-start-loading", handleDidStartLoading);
     webview.addEventListener("did-stop-loading", handleDidStopLoading);
     webview.addEventListener("dom-ready", handleDomReady);
-    webview.addEventListener("did-fail-load", handleFailLoad);
```

Comment:
What's the reason to remove those listeners? If you sure, then remove the const as well.